examples/problems/1000_compute_feature_correlation.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>This is wrong. We don't compute feature corr. between time points, but between push-forward/pull-back distr, and the expression.</p><p>Also what you did here I don't get.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>